raiden/network/rpc/client.py

```diff
@@ -920,64 +925,15 @@ def settle(self):
         # TODO: check if the ChannelSettled event was emitted and if it wasn't raise an error
         log.info('settle called', contract=pex(self.address))
 
-    def channelnewbalance_filter(self):
-        """ Install a new filter for ChannelNewBalance events.
-
-        Return:
-            Filter: The filter instance.
-        """
-        netting_channel_address_bin = self.proxy.address
-        topics = [CHANNELNEWBALANCE_EVENTID]
-
-        filter_id_raw = new_filter(self.client, netting_channel_address_bin, topics)
-
-        return Filter(
-            self.client,
-            filter_id_raw,
-        )
-
-    def channelsecretrevealed_filter(self):
-        """ Install a new filter for ChannelSecret events.
+    def filter_for_all_events(self):
+        """ Install a new filter for the current netting channel contract.
```

Comment:
Install a new filter **for all the events emitted by** the current netting channel contract